Fizz buzz test

OlegKruglov opened the pull request:
Done.

Commit:
Added new branch

Commit:
First test(100 numbers) failure

Commit:
First test(100 numbers) pass

Commit:
Second test (print Fizz instead of every third number) failure

Commit:
Second test (print Fizz instead of every third number) pass

Commit:
Third test(print Buzz instead of every fifth number) failure

Commit:
Third test(print Buzz instead of every fifth number) pass

Commit:
Fourth test(FizzBuzz instead of every third and fifth number) failure

Commit:
Fourth test(FizzBuzz instead of every third and fifth number) pass

kivikangur-tthk commented on FizzBuzzKata/src/FizzBuzzTest.java:
```diff
@@ -6,18 +6,8 @@
-		FizzBuzz fb=new FizzBuzz();
-		assertEquals(100, fb.getNumbers().length);
-	}
+	public void shouldReturn100() {
```
Test name does not reflect it's function

kivikangur-tthk commented on FizzBuzzKata/src/FizzBuzzTest.java:
```diff
@@ -9,5 +9,9 @@
     }
+	
+	public void multipleOfThreeReturnFizz(){
+        assertEquals(FizzBuzz.of(3), ("Fizz"));
```
String does not need parentheses

kivikangur-tthk commented on FizzBuzzKata/src/FizzBuzzTest.java:
```diff
@@ -13,5 +13,9 @@ public void shouldReturn100() {
     }
+	
+	public void multipleOfFiveReturnBuzz(){
+        assertEquals(FizzBuzz.of(5), ("Buzz"));
```
String does not need parentheses

kivikangur-tthk commented on FizzBuzzKata/src/FizzBuzzTest.java:
```diff
@@ -17,5 +17,9 @@ public void multipleOfThreeReturnFizz(){
     }
+	
+	public void multipleOfFiveAndThreeReturnFizzBuzz(){
+        assertEquals(FizzBuzz.of(15), ("FizzBuzz"));
```
String does not need parentheses

kivikangur-tthk commented on FizzBuzzKata/src/FizzBuzz.java:
```diff
@@ -7,6 +7,7 @@ public static void main(String[] args) {
     }
 
     public static String of(int i) {
```
i is not a descriptive name for this variable

kivikangur-tthk reviewed (changes requested):
Fix issues and add me as assignee again.

OlegKruglov committed:
Update FizzBuzzTest.java

OlegKruglov committed:
Update FizzBuzzTest.java

OlegKruglov committed:
Update FizzBuzz.java

OlegKruglov commented:
Fixed issues and added you as assignee.

пн, 13 дек. 2021 г. в 16:52, kivikangur-tthk ***@***.***>:

> Assigned #1
> <https://github.com/OlegKruglov/FizzBuzzKataKruglovTARpv19/pull/1> to
> @OlegKruglov <https://github.com/OlegKruglov>.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OlegKruglov/FizzBuzzKataKruglovTARpv19/pull/1#event-5757441763>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO3FPMCVNBDQ2YGGPAC73NDUQYCB3ANCNFSM5JXLIFGA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

kivikangur-tthk commented on FizzBuzzKata/src/FizzBuzzTest.java:
```diff
@@ -6,20 +6,20 @@
 class FizzBuzzTests {
 
-	public void shouldReturn100() {
+	public void printNumbersFrom1To100() {
```
What does the method do?

OlegKruglov committed:
Update FizzBuzzTest.java
